app/driver/settings.py

```diff
@@ -302,12 +302,19 @@
         'VERSION': 1,
         'OPTIONS': {
             'CLIENT_CLASS': 'django_redis.client.DefaultClient',
-            'SOCKET_CONNECT_TIMEOUT': 5, # seconds
-            'SOCKET_TIMEOUT': 5, # seconds
-            'MAX_ENTRIES': 300, # defaults to 300
-            'CULL_FREQUENCY': 4, # fraction culled when max reached (1 / CULL_FREQ); default: 3
+            'SOCKET_CONNECT_TIMEOUT': 5,  # seconds
+            'SOCKET_TIMEOUT': 5,  # seconds
+            'MAX_ENTRIES': 300,  # defaults to 300
+            'CULL_FREQUENCY': 4,  # fraction culled when max reached (1 / CULL_FREQ); default: 3
             # 'COMPRESS_MIN_LEN': 0, # set to value > 0 to enable compression
         }
+    },
+    "boundaries": {
+        'BACKEND': 'django_redis.cache.RedisCache',
+        'LOCATION': 'redis://{host}:{port}/4'.format(host=REDIS_HOST, port=REDIS_PORT),
+        'TIMEOUT': None,
```

Comment:
We would still have to use `expire()` to renew the expiration each time a cached boundary is accessed so that they only expire if they stop getting used. Given that, I figured it would be easier if all the information about timeouts were contained within one file, rather than relying on a default in `settings.py`, which is why I chose to use `set()` as well. But I don't have strong feelings about it--I could add a comment or a default here if you think having `None` could be confusing or misleading.

Reply:
Comment wouldn't hurt, but I don't think it's a big deal. I'm kind of bummed there doesn't seem to be a way to make it into a LRU cache or have the TTL be timed from the last read, those seem like they wouldn't be an uncommon situation but what we have here works well for what we need.